src/components/pages/ParentDash/components/ChildSignup.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+// Signup form to add children to parent account
+const ChildSignup = () => {
+  return (
+    <form>
```

Comment:
Do you think we should use the Form components form the Ant library here for theme consistency?

Reply:
Yeah I can switch it over, was intended to be a draft but I'll be able to make that more clear with the PR template.